gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/CsekEncryptionIntegrationTest.java

```diff
@@ -72,10 +70,7 @@ public void rewriteObject() throws IOException {
     gcs.createBucket(srcBucketName);
 
     String dstBucketName = BUCKET_HELPER.getUniqueBucketName("rewrite-dst");
-    // Create destination bucket with different location and storage class,
-    // because this is supported by rewrite but not copy requests
-    gcs.createBucket(
-        dstBucketName, CreateBucketOptions.builder().setStorageClass("coldline").build());
+    gcs.createBucket(dstBucketName);
```

Comment:
Can you explain the storage class change?

Reply:
This was copy paste from rewrite requests test where this is necessary, but for CMEK test bucket storage class is irrelevant.